test/integration/consensus/consensus_sunny_day.cpp

```diff
@@ -138,9 +124,13 @@ std::vector<std::shared_ptr<shared_model::interface::Peer>>
     ConsensusSunnyDayTest::default_peers;
 
 TEST_F(ConsensusSunnyDayTest, SunnyDayTest) {
+  std::condition_variable cv;
+  std::mutex m;
   auto wrapper = make_test_subscriber<CallExact>(yac->on_commit(), 1);
-  wrapper.subscribe(
-      [](auto hash) { std::cout << "^_^ COMMITTED!!!" << std::endl; });
+  wrapper.subscribe([&cv](auto hash) {
+    std::cout << "^_^ COMMITTED!!!" << std::endl;
```

Comment:
So ancient and cool.
This is not an issue. Just a warm recall.

Reply:
plz do not delete
it is the only happiness in debugging